### DISCUSSION TITLE
Type Error when running GraphQL api

### DISCUSSION THREAD

**Discussion by causeImCloudy** [requarks/wiki#6662] [Error / Bug Report] [answered]
This is probably pretty low but, when running the below GraphQL api it results in a Type error but successfully updates the page. 

`mutation {
  pages {
    update(
      id: 1,
      content: "Test"
    ) {
      responseResult {
        succeeded
        slug
        message
      }, 
      page{
        id
      }
    }
  }
}
`
Also see the below screenshots. 

<img width="1724" alt="Screenshot 2023-08-19 at 9 43 34 PM" src="https://github.com/requarks/wiki/assets/53097197/361a4421-7844-4185-9c61-506bb9db2dba">

<img width="1728" alt="Screenshot 2023-08-19 at 9 43 15 PM" src="https://github.com/requarks/wiki/assets/53097197/1b4dd86d-d879-490f-9b6b-5d49f371e58a">

**Comment by NGPixel** [comment, marked as answer]
You have to provide all fields, not just content.

This has been fixed in v3.
